scripts/ucsc/template-meta-with-python.yaml

```diff
@@ -23,14 +26,14 @@ requirements:
     - libpng
     - libuuid
     - mysql-connector-c
-    - openssl
+    - libopenssl-static
```

Comment:
interesting why is this needed?

Reply:
The tools require libssl.a instead of libssl.so. I am not sure since when though.
I have seen some recipes using `includes.patch` to change `.a` to `.so`. I decided to keep the original build, i.e. to not patch it, and use the new dependency instead.